syntax = "proto3";

package mldb;

message LibSVMParserConfig {
  // True if feature indices starts from 1.
  bool feature_one_based = 1;

  // True if label indices starts from 1.
  bool label_one_based = 2;
}
